TouchID.android.js

```diff
@@ -1,13 +1,53 @@
-/**
- * Stub of TouchID for Android.
- *
- * @providesModule TouchID
- * @flow
- */
-'use strict';
+import { NativeModules, processColor } from 'react-native';
+const NativeTouchID = NativeModules.FingerprintAuth;
+// Android provides more flexibility than iOS for handling the Fingerprint. Currently the config object accepts customizable title or color. Otherwise it defaults to this constant
 
 export default {
   isSupported() {
-    return false;
+    return new Promise((resolve, reject) => {
+      NativeTouchID.isSupported(
+        error => {
+          return reject(typeof error == 'String' ? createError(error, error) : createError(error));
+        },
+        success => {
+          return resolve(true);
+        }
+      );
+    });
+  },
+
+  authenticate(reason, config) {
+    DEFAULT_CONFIG = { title: 'Authentication Required', color: '#1306ff' };
```

Comment:
why do you not declare the `DEFAULT_CONFIG` variable here?
It causes error in strict mode ?